cypress/e2e/retrievals/download.cy.js

```diff
@@ -101,6 +101,20 @@ describe('Download project spec', () => {
         headers: authHeaders
       }
     )
+    cy.intercept(
```

Comment:
Cant remember from your review, sorry. Where did this extra call to graphql come from?

Reply:
The download page is looping through calls until no items are returned, so this extra intercept returns no items to stop that loop. I was seeing infinite looping calls with the first intercept as a response